raft/src/main/java/org/apache/kafka/raft/RaftConfig.java

```diff
@@ -68,8 +70,21 @@
                 CommonClientConfigs.RETRY_BACKOFF_MS_DOC)
             .define(QUORUM_VOTERS_CONFIG,
                 ConfigDef.Type.LIST,
-                null,
-                new ConfigDef.NonEmptyString(),
+                ConfigDef.NO_DEFAULT_VALUE,
+                new ConfigDef.Validator() {
+                    // Copy from `NonEmptyListValidator` in Connect
```

Comment:
nit: I don't think it's worth mentioning this since the implementation is trivial. If it's really worth worrying about, we could push it into `ConfigDef`.

Reply:
Cool, I will just remove the comment. Having less external change is better.